### PR TITLE
GEOMESA-1595 Docs for bootstrapping via Geodocker

### PR DESCRIPTION
**Pull request by anthonyccri**
* Added documentation for bootstrapping a GeoMesa Accumulo instance
  using AWS EMR and Geodocker

Signed-off-by: Anthony Fox <anthonyfox@ccri.com>

### CODE REVIEW

File: docs/user/aws.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+   DataSourceName: gdelt
+   instanceId: gis
+   zookeepers: $zookeeper
+   user: root
+   password: secret
+   tableName: geomesa.gdelt
+
+Save the store and publish the `gdelt` layer.  Set the 'Native Bounding Box' and the 'Lat Lon Bounding Box' to
+`-180,-90,180,90`.  Save the layer.  Then, navigate to the preview page at `http://<ip_address>:9090/geoserver/cite/wms?service=WMS&version=1.1.0&request=GetMap&layers=cite:gdelt&styles=&bbox=-180,-90,180.0,90&width=768&height=356&srs=EPSG:4326&format=application/openlayers`.
```
Comment (jahhulbert-ccri) [inline docs/user/aws.rst]: does this link work or render as block text? can't tell in github preview.

**Comment by elahrvivaz** [comment]
@anthonyccri was this validated by anyone? are we good to merge?

File: docs/user/aws.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+Bootstrap an EMR cluster
+************************
+Use the following command to bootstrap an EMR cluster.  You will need to change the KeyName to the IAM key pair you intend to use for this cluster.  You can also edit the instance types to a size appropriate for your use case.  Make sure you adjust the Accumulo cache configuration settings accordingly.  For instance, if you use a high memory instance type, you will want to increase the `TSERVER_XMX` parameter which controls the amount of heap space allocated to the JVM running the Accumulo Tablet Server.  Additionally, you should adjust the `TSERVER_CACHE_DATA_SIZE` and the `TSERVER_CACHE_INDEX_SIZE` to appropriate fractions of the `TSERVER_XMX` memory to take advantage of the increase in memory.
```
Comment (mdzimmerman) [inline docs/user/aws.rst]: Unlike Markdown, in RST need double backticks to render something as code ( `TSERVER_XMX` -> ``TSERVER_XMX``). It renders single backticks as italicized.

File: docs/user/aws.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+   sudo docker exec accumulo-master geomesa export -c gdelt -f gdelt -u root -p secret -m 100
+
+You can register GDELT as a layer in the provided geoserver as well.  Geoserver is running on port 9090
+of the master node.  You can access it at `http://<ip_address>:9090/geoserver` where <ip_address> is the
+address you looked up before ssh'ing into the master node.  To register a GeoMesa layer, you'll first need
+to know the internal URL of the zookeeper instance.  Run the following command:
+
+.. code-block:: shell
+   sudo docker exec accumulo-master cat /opt/accumulo/conf/accumulo-site.xml | grep -A2 instance.zoo | grep value | sed 's/.*<value>\(.*\)<\/value>/\1/'
```
Comment (mdzimmerman) [inline docs/user/aws.rst]: Needs to be a blank line between this and the code-block directive.

File: docs/user/aws.rst (hunk 1 above)
Comment (mdzimmerman) [inline docs/user/aws.rst]: No, it doesn't, probably because of the "<ip_address>".

File: docs/user/aws.rst [hunk 4]
```diff
@@ -0,0 +1,103 @@
+=======================================
```
Comment (mdzimmerman) [inline docs/user/aws.rst]: Maybe this could be a tutorial?

File: docs/user/aws.rst [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+      --applications Name=Hadoop Name=Zookeeper Name=Spark \
+      --instance-groups                                    \
+        Name=Master,InstanceCount=1,InstanceGroupType=MASTER,InstanceType=m3.xlarge \
+        Name=Workers,InstanceCount=3,InstanceGroupType=CORE,InstanceType=m3.xlarge  \
+      --bootstrap-actions                                                                        \
+        Name=BootstrapGeoMesa,Path=s3://geomesa-docker/bootstrap-geodocker-accumulo.sh,Args=\[-i=quay.io/geomesa/accumulo-geomesa:latest,-n=gis,-p=secret,-e=TSERVER_XMX=10G,-e=TSERVER_CACHE_DATA_SIZE=6G,-e=TSERVER_CACHE_INDEX_SIZE=2G\]
+
+
+After executing that command, you can monitor the state of the EMR bootstrap process
+by going to the `Management Console <https://console.aws.amazon.com/elasticmapreduce/home?region=us-east-1#cluster-list>`_.  Find the name (as specified in the aws emr command) of the cluster and click through to its details page.  Under the 'Hardware' section, you can find the master node and it's IP address.  Copy the IP address and then run the following command.
```
Comment (mdzimmerman) [inline docs/user/aws.rst]: "it's IP address" -> "its IP address". Also, might want to put aws emr in double backticks (see above).

File: docs/user/aws.rst [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+   5f4ae17263db        quay.io/geomesa/accumulo-geomesa:latest   "/sbin/geomesa-entryp"   5 minutes ago       Up 5 minutes                            accumulo-gc
+   b389e8e45542        quay.io/geomesa/accumulo-geomesa:latest   "/sbin/geomesa-entryp"   5 minutes ago       Up 5 minutes                            accumulo-tracer
+   996327bf6a9f        quay.io/geomesa/accumulo-geomesa:latest   "/sbin/geomesa-entryp"   5 minutes ago       Up 5 minutes                            accumulo-monitor
+   89738108b7a3        quay.io/geomesa/accumulo-geomesa:latest   "/sbin/geomesa-entryp"   5 minutes ago       Up 5 minutes                            accumulo-master
+
+Ingest Public GDELT data
+************************
+
+Make sure you leave enough time for the machine to be completely bootstrapped before running the command to find the docker instances.
+GeoMesa ships with pre-defined data models for many open spatio-temporal data sets such as GDELT.  To ingest the most recent 7 days of `GDELT
```
Comment (mdzimmerman) [inline docs/user/aws.rst]: [nitpicky] "predefined" doesn't need a hyphen.

File: docs/user/aws.rst [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+<http://www.gdeltproject.org>`_ from Amazon's public S3 bucket.
+
+.. code-block:: shell
+
+   FILES=$(seq 7 -1 1 | xargs -n 1 -I{} sh -c "date -d'{} days ago' +%Y%m%d" | xargs -n 1 -I{} echo s3a://gdelt-open-data/events/{}.export.csv | tr '\n' ' ')
+   sudo docker exec accumulo-master geomesa ingest -c geomesa.gdelt -C gdelt -f gdelt -s gdelt -u root -p secret $FILES
+
+You can then query the data using GeoMesa command line export tool.
```
Comment (mdzimmerman) [inline docs/user/aws.rst]: "*the* GeoMesa command line"?

File: docs/user/aws.rst [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+.. code-block:: shell
+
+   sudo docker exec accumulo-master geomesa export -c gdelt -f gdelt -u root -p secret -m 100
+
+You can register GDELT as a layer in the provided geoserver as well.  Geoserver is running on port 9090
```
Comment (mdzimmerman) [inline docs/user/aws.rst]: [extra nitpicky] "[gG]eoserver" -> "GeoServer"

File: docs/user/aws.rst [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+      --applications Name=Hadoop Name=Zookeeper Name=Spark \
+      --instance-groups                                    \
+        Name=Master,InstanceCount=1,InstanceGroupType=MASTER,InstanceType=m3.xlarge \
+        Name=Workers,InstanceCount=3,InstanceGroupType=CORE,InstanceType=m3.xlarge  \
+      --bootstrap-actions                                                                        \
+        Name=BootstrapGeoMesa,Path=s3://geomesa-docker/bootstrap-geodocker-accumulo.sh,Args=\[-i=quay.io/geomesa/accumulo-geomesa:latest,-n=gis,-p=secret,-e=TSERVER_XMX=10G,-e=TSERVER_CACHE_DATA_SIZE=6G,-e=TSERVER_CACHE_INDEX_SIZE=2G\]
+
+
+After executing that command, you can monitor the state of the EMR bootstrap process
+by going to the `Management Console <https://console.aws.amazon.com/elasticmapreduce/home?region=us-east-1#cluster-list>`_.  Find the name (as specified in the ``aws emr`` command) of the cluster and click through to its details page.  Under the **Hardware** section, you can find the master node and its IP address.  Copy the IP address and then run the following command.
```
Comment (jnh5y) [inline docs/user/aws.rst]: Does it matter that us-east-1 is mentioned here?

**Comment by jnh5y** [comment]
System.getenv("ZOOKEEPERS") @jnh5y

**Comment by jnh5y** [comment]
Closing in favor of https://github.com/locationtech/geomesa/pull/1334

**Pull request closed without merging**